graylog2-server/src/test/java/org/graylog2/inputs/codecs/SyslogCodecTest.java

```diff
@@ -426,8 +427,18 @@ public void testCiscoSyslogMessages() {
     }
 
     @Test
-    public void testFortiGateFirewall() {
-        final RawMessage rawMessage = buildRawMessage("<45>date=2017-03-06 time=12:53:10 devname=DEVICENAME devid=DEVICEID logid=0000000013 type=traffic subtype=forward level=notice vd=ALIAS srcip=IP srcport=45748 srcintf=\"IF\" dstip=IP dstport=443 dstintf=\"IF\" sessionid=1122686199 status=close policyid=77 dstcountry=\"COUNTRY\" srccountry=\"COUNTRY\" trandisp=dnat tranip=IP tranport=443 service=HTTPS proto=6 appid=41540 app=\"SSL_TLSv1.2\" appcat=\"Network.Service\" applist=\"ACLNAME\" appact=detected duration=1 sentbyte=2313 rcvdbyte=14883 sentpkt=19 rcvdpkt=19 utmaction=passthrough utmevent=app-ctrl attack=\"SSL\" hostname=\"HOSTNAME\"");
+    public void testFortiGate() {
+        doTestFortigate(FORTIGATE);
+    }
+
+    @Test
+    public void testFortiGateTrimLineBreaks() {
+        // Ensure that trailing line breaks are trimmed for Fortigate messages to avoid parsing error.
+        doTestFortigate(FORTIGATE + "\r\n ");
+    }
+
+    private void doTestFortigate(String FORTIGATE) {
+        final RawMessage rawMessage = buildRawMessage(FORTIGATE);
```

Comment:
nit: capitalization, also naming of variable
```suggestion
    private void doTestFortigate(String message) {
        final RawMessage rawMessage = buildRawMessage(message);
```

Reply:
Oops, thanks for catching that.